extra/systemd/foreman-proxy.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Foreman Proxy
+Wants=basic.target
+After=basic.target network.target
+
+[Service]
+Type=notify
+User=foreman-proxy
+ExecStart=/usr/share/foreman-proxy/bin/smart-proxy --no-daemonize
+EnvironmentFile=-/etc/sysconfig/foreman-proxy
```

Comment:
Good point, I had been thinking that packages for other OSes could patch the location, but preferring the systemd overrides seems a much better default. This can be added back in for RPM packages for their backwards-compatibility. Line removed.